crates/redeem/src/lib.rs

```diff
@@ -669,7 +674,7 @@ impl<T: Config> Pallet<T> {
 
         let fee = satoshi_per_bytes
             .checked_mul_int(size)
-            .ok_or(Error::<T>::ArithmeticOverflow)?;
+            .ok_or(ArithmeticError::Overflow)?;
```

Comment:
We'd lose the information about in which module this error occurred, right? Not sure if that'd be a big problem, but something to consider

Reply:
We anyway lost this with the currency module upgrade FWIW.

Reply:
Hmm that's fair